es6/experiment.js

```diff
@@ -100,7 +100,20 @@ class Experiment {
   }
 
   assign(params, args) {
-    throw "IMPLEMENT THIS";
+    throw "IMPLEMENT assign";
+  }
+
+  experimentParameters() {
```

Comment:
yes that's much a much better name for this function. Naming things is hard :). I've been meaning to experiment with adding Flow to this project, but have not gotten around to it quite yet. 

Do you think we should add something similar to the Python implementation?